src/languages/python.js

```diff
@@ -176,7 +176,7 @@ export default function(hljs) {
     contains: [ hljs.BACKSLASH_ESCAPE ],
     variants: [
       {
-        begin: /([uU]|[bB]|[rR]|[bB][rR]|[rR][bB])?'''/,
+        begin: /(\W|^)([uU]|[bB]|[rR]|[bB][rR]|[rR][bB])?'''/,
```

Comment:
Aren't you really just expressing "word boundary" here... and can't that be done much simpler using `\b`?

Reply:
I think it's doing the same thing here. 
But it didn't come first to my mind. 
Thanks for your advice and I will replace `(\W|^)` with it.

Reply:
@joshgoebel please take another look at my changes.
Many thanks for considering my request.